src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS/IISDeployer.cs

```diff
@@ -364,6 +365,7 @@ private void Stop()
                                 wp.State == WorkerProcessState.Running ||
                                 wp.State == WorkerProcessState.Stopping))
                         {
+                            Logger.LogInformation($"WorkerProcess not stopped yet");
```

Comment:
Why do we need this in addition to exceptions.

Reply:
Was useful for debugging why the worker process wasn't being killed.